.circleci/config.yml

```diff
@@ -57,10 +67,8 @@ workflows:
             - csharp-app-encryption
       - go-securememory
       - go-appencryption
-      - go-referenceapp:
-          requires:
-            - go-appencryption
-      - cross-language:
+      - samples-go-reference-app
```

Comment:
nit: good callout on renaming it. but it should be `samples-go-referenceapp`

Reply:
followed the same naming convention as the jobs for Java and C#. seemed better to keep the naming scheme consistent within the config file